### DISCUSSION TITLE
Control panel UI for static translations

### DISCUSSION THREAD

**Discussion by thaoms** [craftcms/cms#8488] [Ideas]
Hello

We let our clients fill in their own translations. (Belgium has three official languages)
That often includes static translations.

We can't just send them php files so I was wondering if this ever gets included in the control panel?
A simple UI is much more clear for less technical clients.

There used to be an old plugin for Craft 2, but this has been abandoned a while ago.
(https://github.com/boboldehampsink/translate)

I am using globals to simulate this now, but I guess that's not why globals were made.

Thank you!

**Comment by brandonkelly** [comment]
I could see this being a Utility. Though UI copy is a perfectly valid use case for globals :)

The main reason static translations exist is for Control Panel copy (for Craft and plugins), which doesn’t require an optimized author-facing UI for being customized.

**Comment by outline4** [comment]
In Switzerland we have 4 official Languages. There's a real need for static translation that can be edited in the CP. Globals work, but there's too much work involved to set it up...

So a utility would be really great! 

Thanks

**Comment by smcyr** [comment]
It would be nice, because if you have hundred of strings to translate across the website, if you do a field per strings in the globals, the database will have too many columns in the "content" table.
Thanks

**Comment by thaoms** [comment]
I stumbled upon this one, didn't use it yet but it looks good.
https://enupal.com/en/craft-plugins/translate

**Comment by bertheyman** [comment]
This would really be a nice feature!
Someone created a [plugin](https://github.com/MutationDigitale/craft3-translate), but would prefer this as a native Craft feature. (ongoing support + make sure it keeps up with other new Craft features)

**Comment by jan-dh** [comment]
One tricky part about this: the translations are being handles on a per-language base. Let's say you have a multi-site setup, 3 websites that all share the same language and 3 editor groups who can all edit only 1 site. If the translation on site 1 is being altered, it will also change for site 2 and 3. It would be better if we could pass another argument to the translation filter to link to a site/sitegroup to have more control on these translations.

**Comment by alexanderbuergin** [comment]
+1
